requirements.txt

```diff
@@ -1,4 +1,4 @@
--e git+git@github.com:rossnanop/pyechonest.git@c8b1106e5308c75a88d8195557aad5e6e82d7d7a#egg=pyechonest-master
+-e git+git@github.com:https://github.com/echonest/pyechonest.git@c8b1106e5308c75a88d8195557aad5e6e82d7d7a#egg=pyechonest-master
```

Comment:
The structure of this URI looks wrong to me now and it gives me the error below. Here's the pip documentation on what git URIs it supports: https://pip.pypa.io/en/stable/reference/pip_install/#git. Perhaps it seemed like this was working for you because you ran `pip install -r requirements.txt` in a virtualenv which already had echonest installed? I don't know why I had "rossnanop", perhaps his PR had not merged into the main echonest repo a few weeks ago. Shall we change it to "echonest"? Does the `git+git` protocol specification work for you?

```
/tmp/music-graph() ./venv/bin/pip install -r requirements.txt
Obtaining pyechonest-master from git+git@github.com:https://github.com/echonest/pyechonest.git@c8b1106e5308c75a88d8195557aad5e6e82d7d7a#egg=pyechonest-master (from -r requirements.txt (line 1))
  Cloning git@github.com:https://github.com/echonest/pyechonest.git (to c8b1106e5308c75a88d8195557aad5e6e82d7d7a) to ./venv/src/pyechonest-master
fatal: I don't handle protocol 'git@github.com:https'
  Complete output from command git clone -q git@github.com:https://github.com/echonest/pyechonest.git /private/tmp/music-graph/venv/src/pyechonest-master:

  ----------------------------------------
Command "git clone -q git@github.com:https://github.com/echonest/pyechonest.git /private/tmp/music-graph/venv/src/pyechonest-master" failed with error code 128 in None
```